Formula/emqx.rb

```diff
@@ -21,14 +21,14 @@ class Emqx < Formula
   depends_on "ccache"    => :build
   depends_on "cmake"     => :build
   depends_on "coreutils" => :build
-  depends_on "erlang@24" => :build
+  depends_on "erlang@25" => :build
```

Comment:
cc @id 
```suggestion
  depends_on "erlang" => :build
```

Reply:
@chenrui333 there is a risk that emqx build will break with the next major erlang release (26, should be in May or June this year). It would be safer to explicitly pin the erlang version to 25.

Reply:
We will regression test all the dependants when we bump erlang to v26, so that should not be a concern.